filebeat/harvester/util.go

```diff
@@ -2,6 +2,18 @@ package harvester
 
 import "github.com/elastic/beats/libbeat/common/match"
 
+// Contains available prospector types
+const (
+	LogType   = "log"
+	StdinType = "stdin"
+)
+
+// List of valid input types
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported var ValidType should be of the form "ValidType ..."